desktop/src/main/java/bisq/desktop/components/paymentmethods/RevolutForm.java

```diff
@@ -73,7 +75,6 @@ public void addFormForAddAccount() {
         userNameInputTextField = FormBuilder.addInputTextField(gridPane, ++gridRow, Res.get("payment.account.userName"));
         userNameInputTextField.setValidator(validator);
         userNameInputTextField.textProperty().addListener((ov, oldValue, newValue) -> {
-            account.setUserName(newValue.trim());
             updateFromInputs();
```

Comment:
I think the problem is that the setUserName method has side effects. I will make an alternative PR with solving the problem at the domain layer. Will be also more useful once used from API.